ironfish-cli/src/commands/airdrop/constants.ts

```diff
@@ -0,0 +1,5 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+export const AIRDROP_NOTES_IN_BLOCK = 600
```

Comment:
`AIRDROP_NOTES_PER_TRANSACTION` might be clearer, but again, don't mind it too much

Reply:
We can update this in later PR, I will keep a list of notes for what we want to do after getting these merged. I just don't want to go through all the rebases again.